vitrina/fields.py

```diff
@@ -252,3 +255,41 @@ class DisabledTextInput(TextInput):
 
 class DisabledCharField(CharField):
     widget = DisabledTextInput
+
+
+class StringListWidget(Widget):
+    template_name: str = "component/multi_input.html"
+    validation_errors: list[str | None]
+
+    def value_from_datadict(
+        self, data: QueryDict, files: MultiValueDict, name: str
+    ) -> list[str]:
+        values = data.getlist(name)
+        return [v.strip() for v in values if v.strip()]
+
+    def get_context(
+        self, name: str, value: Sequence[str] | None, attrs: Mapping[str, Any] | None
+    ) -> dict[str, Any]:
+        context = super().get_context(name, value, attrs)
+        values = value or []
+        validation_errors = getattr(self, "validation_errors", [None for v in values])
```

Comment:
Not using anything from the loop, just use the underscore:

```suggestion
        validation_errors = getattr(self, "validation_errors", [None for _ in values])
```